crm_sector/models/crm_lead.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, exceptions, _
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    sector_id = fields.Many2one(comodel_name='res.partner.sector',
+                                string='Main Sector')
+
+    secondary_sector_ids = fields.Many2many(
+        comodel_name='res.partner.sector', string="Secondary Sectors",
+        domain="[('id', '!=', sector_id)]")
+
+    @api.constrains('sector_id', 'secondary_sector_ids')
+    def _check_sectors(self):
+        if self.sector_id in self.secondary_sector_ids:
+            raise exceptions.Warning(_('The main sector must be different '
```

Comment:
actually this is the other way around:
'The secondary sectors must be different '
'from the main sector.'